content/applications/productivity/discuss/overview/team_communication.rst

```diff
@@ -97,3 +97,18 @@ drop-down menu.
 .. seealso::
    - :doc:`get_started`
    - :doc:`plan_activities`
+
+Linking channel in chatter
+==========================
+
+Channels can be linked in the chatter (log note) of a record in Odoo. To do so, simply type: `#` and
+the channel name. Click or press enter on the *channel* name. Upon logging the note a link to the
+channel will appear. After clicking on the link a chat window with the channel conversation will
+pop-up in the lower right corner of the screen.
```

Comment:
**pop-up** is a compound adjective
**pop up** is the verb you're looking for ;)
```suggestion
pop up in the lower right corner of the screen.
```